### PR TITLE
Format line lengths with golines

### PR DESCRIPTION
**Pull request by djjuhasz**

### CODE REVIEW

**Comment by jraddaoui** [comment]
Nice, I still need to add this to other repos too.

**Review by jraddaoui — APPROVED** [review]

**Comment by djjuhasz** [comment]
To avoid having to reformat all the commits in PR #5 I'll wait until #5 is merged, then run golines again on this branch, then merge this PR.

**Pull request merged**
